client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const _ = require("lodash");
```

Comment:
It turns out that the polyfill is not necessary for our test suite to pass and browsers we currently support should be fine with `Object.assign` used in https://github.com/thelounge/lounge/pull/972/commits/cc7d121c03e95e0554e6fe81dbc066fbea9cd7cb.